### PR TITLE
Sign file bugfix #133

### PR DESCRIPTION
**Pull request by koudis**
Bugfix with "multiple sign certification" (#133) can be cleanly
solve by specifying path to sign cert.

### CODE REVIEW

File: driver/vhci/gencat.bat [hunk 1]
```diff
@@ -6,7 +6,7 @@ copy ..\usbip_vhci.sys
 copy ..\usbip_vhci.cer
 copy ..\usbip_vhci.inf
 inf2cat /driver:.\ /os:%2 /uselocaltime
-signtool sign usbip_vhci.cat
+signtool sign /f %3 /p usbip usbip_vhci.cat
```
Comment (koudis) [inline driver/vhci/gencat.bat:9]: I store password directly to bat file. If needed i can move it as a script parameter.

Comment (bartlomiejcieszkowski) [reply driver/vhci/gencat.bat:9]: thats ok by me also, as it uses the file from repository, so thats good for potential ci/cd

**Comment by koudis** [comment]
OK, I will merge :)

**Comment by koudis** [comment]
@cezanne merged

**Pull request merged**